### PR TITLE
Feat/feeders exchange label

### PR DESCRIPTION
**Pull request by ghigheci**
implemented tests for PrivateKrakenFeeder and PublicKrakenFeeder

added exchange and symbol labels for trade, order, book and candle events

### CODE REVIEW

File: src/tradingkit/data/feed/backtest_feeder.py [hunk 1]
```diff
@@ -50,4 +50,5 @@ def dispatch_month(self, exchange, symbol, year, month, since, to):
 
         for trade in json.load(open(full_filename, 'r')):
             if since.timestamp() <= trade['timestamp'] / 1000 < to.timestamp():
+                trade['exchange'] = self.exchange
```
Comment (santoslluis) [inline src/tradingkit/data/feed/backtest_feeder.py:53]: self.exchange is a object or string?

event data must be serializable

File: src/tradingkit/data/feed/kraken_feeder.py [hunk 2]
```diff
@@ -26,13 +26,15 @@ class KrakenFeeder(Feeder, Publisher):
             "BTC/EUR": "XBT/EUR",
             "BTC/USD": "XBT/USD",
             "BTC/USDT": "XBT/USDT",
+            "ETH/BTC": "ETH/XBT",
```
Comment (santoslluis) [inline src/tradingkit/data/feed/kraken_feeder.py:29]: must create separate PR's for different change types

**Review by santoslluis — CHANGES_REQUESTED** [review]

File: src/tradingkit/data/feed/backtest_feeder.py (hunk 1 above)
Comment (ghigheci) [reply src/tradingkit/data/feed/backtest_feeder.py:53]: is a string

File: src/tradingkit/data/feed/kraken_feeder.py (hunk 2 above)
Comment (ghigheci) [reply src/tradingkit/data/feed/kraken_feeder.py]: done

**Pull request merged**